### PR TITLE
typo

### PR DESCRIPTION
**Pull request by DennisYurichev**
Small typo

### CODE REVIEW

File: src/sage/logic/logictable.py [hunk 1]
```diff
@@ -62,7 +62,7 @@
     True   True   False  True
     True   True   True   True
 
-Get the letex code for a truth table::
+Get the latex code for a truth table::
```
Comment (mantepse) [inline src/sage/logic/logictable.py]: LaTeX seems more common in the documentation, I'd suggest to use that instead.

Found using
```grep -r --include=*.{py,pyx} --color -nH --null -e "[^.(_]LaTeX[^_('\`\"=]" *```
and
```grep -r --include=*.{py,pyx} --color -nH --null -e "[^.(_]latex[^_('\`\"=]" *```

**Comment by DennisYurichev** [comment]
So? Should I do something? Please advise.

**Comment by grhkm21** [comment]
Yes, replace "latex" with "LaTex".

**Comment by DennisYurichev** [comment]
OK, did so: https://github.com/sagemath/sage/pull/37688/commits/3b137b50a0112ef56ac7fa63c64cb0a1c5671db5

**Review by mantepse — APPROVED** [review]
Thank you!

**Pull request merged**